### PR TITLE
Issue/372/v1.0 code formatting

### PR DESCRIPTION
**Pull request by m-aguena**
Final formatting for v1.0

### CODE REVIEW

File: clmm/utils.py [hunk 1]
```diff
@@ -159,35 +172,39 @@ def convert_units(dist1, unit1, unit2, redshift=None, cosmo=None):
...
         # Make sure that we were passed a redshift and cosmology
         if redshift is None or cosmo is None:
-            raise TypeError("Redshift and cosmology must be specified to convert units")
+            raise TypeError(
+                "Redshift and cosmology must be specified to convert units") \
+                from u.UnitConversionError
         # Redshift must be greater than zero for this approx
         if not redshift > 0.0:
-            raise ValueError("Redshift must be greater than 0.")
+            raise ValueError("Redshift must be greater than 0.") from u.UnitConversionError
```
Comment (combet) [inline clmm/utils.py:185]: What does the `from u.UnitConversionError` adds here, compared to what was there previously? Just out of curiosity as I haven't seen this used before.

Comment (combet) [reply clmm/utils.py:185]: Forget about it, just tried it and saw the improved print out clearly indicating the type of error as `UnitConversionError` when using a bad redshift.

**Review by combet — APPROVED** [review]
Thanks @m-aguena. All looks good, with consistent formatting throughout the code base!

**Review by combet — DISMISSED** [review]
@m-aguena - just noticed going through the notebooks that 
- the outputs of `demo_theory_functionality.ipynb` and  `demo_theory_functionality_oo.ipynb` need to be cleared (must be reminiscent from somwhere else)
- Cell 5 of `demo_theory_functionality_oo.ipynb` has 
`moo = clmm.Modeling(massdef='mean', delta_mdef=200, halo_profile_model='hernquist')`
but it should be
`moo = clmm.Modeling(massdef='mean', delta_mdef=200, halo_profile_model='nfw')` to make sure it works with all backends. (Again likely reminiscent from previous tests that where not reverted)

**Comment by combet** [comment]
Actually I had some time now, so took care of this final small point

**Comment by combet** [comment]
From #372 that there are still two boxes to tick related to that issue/PR.
- [ ] Fix formatting of docstring for `z_src_model` in `clmm/theory/func_layer.py`
- [ ] Fix equation of Sigma in documentation to `\int \rho`

**Comment by m-aguena** [comment]
@combet both should be fixed now

**Review by combet — APPROVED** [review]
Thanks a lot @m-aguena!

**Pull request merged**